src/pipeline/src/etl.rs

```diff
@@ -210,6 +210,37 @@ where
         self.transformer.transform_mut(val)
     }
 
+    pub fn prepare_pipeline_value(&self, val: Value, result: &mut [Value]) -> Result<()> {
```

Comment:
Is the original `pub fn prepare(&self, val: serde_json::Value, result: &mut [Value]) -> Result<()> {`  deprecated?

Reply:
`prepare` is transform serde_json::Value
this method is transform pipeline Value